doc/design/cluster_train/submit-job.md

```diff
@@ -0,0 +1,111 @@
+# Submit a Distributed Training Job
+
+If a user wants to start up a distributed training job, he will submit the distributed training job with Python code.
+
+If a user wants to start up a local train, he will start up a PaddlePaddle production Docker container firstly, and then
+execute `python train.py` in the Docker container. The details about PaddlePaddle Docker image is [here](../../../paddle/scripts/docker/README.md).
+
+## Runtime Environment On Kubernetes
+
+For a distributed training job, there is two Docker image called *runtime Docker image* and *base Docker image*. The runtime Docker image is the Docker image that gets scheduled by Kubernetes to run during training. The base Docker image is for building the runtime Docker image.
+
+- Base Docker Image
+
+  Usually, the base Docker image is PaddlePaddle product Docker image including paddle binary files and trainer startup script file. And of course, users can specify any image name hosted on any docker registry which users have the access right.
+
+- Runtime Docker Image
+
+  The trainer package which user upload and some Python dependencies are packaged into a runtime Docker image based on base Docker image
+
+- Python Dependencies
+
+  You need to provide requirments.txt file in your "trainer" package. Example:
+  ```txt
+  pillow
+  protobuf==3.1.0
+  ```
+  More [details](https://pip.readthedocs.io/en/1.1/requirements.html) about requirements.
+
+  An example project looks like:
+  ```bash
+    paddle_example
+      |-quick_start
+        |-trainer.py
+        |-dataset.py
+        |-requirments.txt
+  ```
+
+## Submit Distributed Training Job With Python Code
+<img src="./src/submit-job-python.png" width="800">
```

Comment:
Need a paragraph to describe the flow in a big picture. The paragraph below goes directly to the detail of `paddle.dist_train`, the reader needs a big picture to follow the concept.